usage/rest-client/src/main/java/brooklyn/rest/client/BrooklynApi.java

```diff
@@ -170,7 +169,7 @@ public AccessApi getAccessApi() {
         return proxy(AccessApi.class);
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({ "unchecked", "rawtypes" })
```

Comment:
Better use a wildcard in the cast, no need for any `@SuppressWarnings` then, i.e.
```
ClientResponse<?> clientResponse = (ClientResponse<?>) response;
```

Same for next method.